nodes/Lambda.py

```diff
@@ -1 +1,17 @@
-fun = lambda: 3
+"""
+Lambda astroid node
+
+Lambda is a minimal function definition that can be used inside an expression.
+
+Attributes:
+    - args  (list)
+        - The arguments for function lambda.
+    - body  (node)
+        - The body of function lambda.
```

Comment:
It would be good to highlight here that the body is a single node, not a list of nodes; this is pretty interesting.